subxt/src/client.rs

```diff
@@ -264,21 +294,63 @@ where
                 .system_account_next_index(signer.account_id())
                 .await?
         };
-        let call = self
-            .client
-            .metadata()
-            .pallet(C::PALLET)
-            .and_then(|pallet| pallet.encode_call(&self.call))?;
-
-        let signed = extrinsic::create_signed(
-            &self.client.runtime_version,
-            self.client.genesis_hash,
+
+        // 2. SCALE encode call data to bytes (pallet u8, call u8, call params).
+        let call_data = {
+            let mut bytes = Vec::new();
+            let pallet = self.client.metadata().pallet(C::PALLET)?;
+            bytes.push(pallet.index());
+            bytes.push(pallet.call_index::<C>()?);
+            self.call.encode_to(&mut bytes);
+            Encoded(bytes)
+        };
+
+        // 3. construct our custom additional/extra params.
+        let additional_and_extra_params = X::new(
+            self.client.runtime_version.spec_version,
+            self.client.runtime_version.transaction_version,
             account_nonce,
-            call,
-            signer,
-            additional_params,
-        )
-        .await?;
-        Ok(signed)
+            self.client.genesis_hash,
+            other_params,
+        );
+
+        // 4. construct signature
```

Comment:
Please provide explanation and maybe a link that this is a copy of the `Encode` impl for `SignedPayload`